fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/sfcc.rs

```diff
@@ -385,46 +358,13 @@ impl SlatFlapControlComputer {
     }
 
     fn slat_actual_position_word(&self) -> Arinc429Word<f64> {
-        Arinc429Word::new(
-            self.slats_feedback_angle.get::<degree>(),
-            SignStatus::NormalOperation,
-        )
+        let slats_fppu = self.slats_channel.get_feedback_angle();
```

Comment:
```suggestion
        let slats_angle = self.slats_channel.get_feedback_angle();
```

Reply:
Did you mean `fppu_slaps_angle` as line 298?

Reply:
Either or should work. I think the `fppu` part is not that important (also not on line 298 actually).